### PR TITLE
74 success add activity page

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#74 
---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -69,6 +69,18 @@ function App() {
                 headerIcon="darts"
                 goBack
               />
+
+              {/* HQ routes */}
+
+              <Route
+                exact
+                // path={navRoutes.HQ.SUCCESS_UPDATE_ACTIVITY}
```
Comment (thejoefriel) [inline client/src/App.js:77]: /test was very useful for testing :) But can you change to the right path now please

File: client/src/components/Icon/icons/Eay.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const Eay = ({ width, height, color, ...props }) => (
```
Comment (thejoefriel) [inline client/src/components/Icon/icons/Eay.js:1]: Is this the eye symbol by view activity? If so can you rename this icon 'eye'

**Review by thejoefriel — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (fadeomar) [reply client/src/App.js:77]: done

File: client/src/components/Icon/icons/Eay.js (hunk 2 above)
Comment (fadeomar) [reply client/src/components/Icon/icons/Eay.js:1]: done

**Review by thejoefriel — APPROVED** [review]

**Pull request merged**